plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergTableProperties.java

```diff
@@ -45,13 +48,29 @@ public class IcebergTableProperties
     public static final String FORMAT_VERSION_PROPERTY = "format_version";
     public static final String ORC_BLOOM_FILTER_COLUMNS = "orc_bloom_filter_columns";
     public static final String ORC_BLOOM_FILTER_FPP = "orc_bloom_filter_fpp";
+    public static final String EXTRA_PROPERTIES = "extra_properties";
+
+    public static final String TABLE_COMMENT = "comment";
```

Comment:
`TABLE_COMMENT` is a "native" Iceberg table property and not a Trino table property.

Trino offers `COMMENT` syntax for such a purpose.

Pls remove.

Reply:
I believe there is some misunderstanding here because I am adhering to the definition in the HiveMetadata class and moving it here, since I believe it doesn't make sense to import certain properties from HiveMetadata. If it's wrong here, it's also wrong in HiveMetadata.

Currently orc.bloom.filter.columns and orc.bloom.filter.fpp are defined in IcebergMetadata, while comment comes from HiveMetadata.

The comment property is not native to Iceberg, as confirmed by the Iceberg TableProperties link: [Iceberg TableProperties](https://github.com/apache/iceberg/blob/main/core/src/main/java/org/apache/iceberg/TableProperties.java) and as discussed in this [Iceberg Slack thread](https://apache-iceberg.slack.com/archives/C03LG1D563F/p1706217171665909)

My suggestion is to maintain the TABLE_COMMENT here and create a set including TABLE_COMMENT, ORC_BLOOM_FILTER_COLUMNS, ORC_BLOOM_FILTER_FPP, DEFAULT_FILE_FORMAT, and FORMAT_VERSION, as imported from org.apache.iceberg.TableProperties, to address the issue @oneonestar mentioned in another comment.

What do you think?

Reply:
Thinking about it again, I believe that the 'comment' constant should not reside in HiveMetadata nor in IcebergTableProperties, because this is a reserved Trino property. But I don't know where the better place would be. Any thoughts?